src/pages/Schedule/index.tsx

```
@@ -200,159 +236,266 @@ const SchedulePage = () => {
+                          return Promise.reject('Start time cannot be after end time!')
+                        }
+                        return Promise.resolve()
+                      }
+                    })
+                  ]}
+                >
+                  <TimePicker
+                    format={'HH:mm:ss'}
+                    placeholder='Start Time'
+                    style={{ width: '100%' }}
+                    disabledTime={(current) => {
```

ESLint

[error] 'current' is defined but never used.

(@typescript-eslint/no-unused-vars)